understanding/name-role-value.html

```diff
@@ -58,106 +56,79 @@
                   
-                  <dt>Why it's important</dt>
-                  <dd>Assistive technology only works well when code is done properly.</dd> 
+                  <dt>なぜそれが重要か</dt>
+                  <dd>支援技術は、コードが適切に記述されている場合にのみうまく機能する。</dd> 
                   
                </dl>
             </section>
             <section id="intent">
-               <h2>Intent</h2>
-               <p>The intent of this Success Criterion is to ensure that Assistive Technologies (AT)
-                  can gather appropriate information about, activate (or set) and keep up to date on
-                  the status of
-                  user interface controls in the content.
+               <h2>意図</h2>
+               <p>この達成基準の意図は、支援技術がコンテンツにあるユーザインタフェース コントロールの状態に関する適切な情報を収集し、有効化 (又は設定) 及び最新の状態を保つように保証することである。
                   
                </p>
-               <p>When standard controls from accessible technologies are used, this process is straightforward.
-                  If the user interface elements are used according to specification the conditions
-                  of this provision will be met. (See examples of Success Criterion 4.1.2 below)
+               <p>アクセシブルなウェブコンテンツ技術の標準コントロールを使用する場合、このプロセスは簡単である。ユーザインタフェース要素が仕様に準じて使用される場合、この条件に条項は満たされる。(下記の達成基準 4.1.2 の事例を参照のこと)
                   
                </p>
-               <p>If custom controls are created, however, or interface elements are programmed (in
-                  code or script) to have a different role and/or function than usual, then additional
-                  measures need to be taken to ensure that the controls provide important and appropriate
-                  information
-                  to assistive technologies and allow themselves to be controlled by assistive technologies.
+               <p>しかし、カスタムコントロールが作成される、又はインタフェース要素が通常とは異なる役割及び／又は機能を持つように (ソースコード又はスクリプトで) プログラミングされている場合、コントロールが支援技術に重要及び適切な情報を提供し、支援技術によってコントロールされることを、それら自身ができるように保証する追加の手段を取る必要がある。
                   
                </p>
-               <p>What roles and states are appropriate to convey to assistive technology will depend
-                  on what the control represents. Specifics about such information are defined by other
-                  specifications, such as <a href="https://www.w3.org/TR/wai-aria/">WAI-ARIA</a>, or the
-                  relevant platform standards. Another factor to consider is whether there is sufficient
-                  <a href="https://www.w3.org/TR/WCAG22/#dfn-accessibility-supported">accessibility support</a> 
-                  with assistive technologies to convey the information as specified.
+               <p>どのような役割 (role) 及び状態を支援技術に伝えるのが適切かは、コントロールが何を表すかによる。そのような情報に関する詳細は、<a href="https://www.w3.org/TR/wai-aria/">WAI-ARIA</a> などの他の仕様、関連するプラットフォーム標準によって定義される。考慮すべきもう一つの要素は、指定されたとおりに情報を伝達するために、支援技術とともに十分な<a href="https://waic.jp/translations/WCAG22/#dfn-accessibility-supported">アクセシビリティサポート</a>があるかどうかである。
```

Comment:
as specified は「仕様の通りに」という意味だと思います。仕様的にはOKだけどASではない場合がある、という文脈の話なので。あと「アクセシビリティ サポート」はスペースが入るでしょうか。

```suggestion
               <p>どのような役割 (role) 及び状態を支援技術に伝えるのが適切かは、コントロールが何を表すかによる。そのような情報に関する詳細は、<a href="https://www.w3.org/TR/wai-aria/">WAI-ARIA</a> などの他の仕様、関連するプラットフォーム標準によって定義される。考慮すべきもう一つの要素は、仕様のとおりに情報を伝達するような、支援技術の十分な<a href="https://waic.jp/translations/WCAG22/#dfn-accessibility-supported">アクセシビリティ サポート</a>があるかどうかである。
```